launch/testnet/start-node.sh

```diff
@@ -0,0 +1,5 @@
+go build
+
+# Launch processes in the background, and ignore their output
+./blindbid &>/dev/null &
+./testnet &>/dev/null &
```

Comment:
in case of an unrecoverable error, the latter should be trapped, logged and the script should return an error code on exit

Reply:
The script runs the executables in the background, so that control is returned to the caller instantly (to then, for example, open the wallet). So the exit code can not be returned from the script itself.

In case of a panic though, maybe it makes sense to replace the `panic` statements and log these instead with `log.Panic`, so that they show up in the logfile.